UserAgent.xcodeproj/project.pbxproj

```diff
@@ -5615,7 +5615,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export NODE_BINARY=`asdf which node`\n./node_modules/react-native/scripts/react-native-xcode.sh\n";
+			shellScript = "export NODE_BINARY=`readlink -f ~/.asdf/shims/node`\n\n./node_modules/react-native/scripts/react-native-xcode.sh\n";
```

Comment:
No problem for me, but note that "readlink -f" is a GNU extension. Out of the box, not available on MacOS.

I have not tested it, but "realpath" might be the safer way on MacOS. (On Linux, it is also there.)

Reply:
thanks for suggestion!